crates/nu-command/src/commands/core_commands/tags.rs

```diff
@@ -18,6 +18,10 @@ impl WholeStreamCommand for Tags {
         "Read the tags (metadata) for values."
     }
 
+    fn extra_usage(&self) -> &str {
```

Comment:
Can we use a default that outputs the empty string instead of adding this to all the commands?